basis/formatting/formatting.factor

```diff
@@ -113,9 +113,9 @@ fmt-C     = "C"                  => [[ [ 1string >upper ] ]]
 fmt-s     = "s"                  => [[ [ present ] ]]
 fmt-S     = "S"                  => [[ [ present >upper ] ]]
 fmt-u     = "u"                  => [[ [ unparse ] ]]
-fmt-d     = "d"                  => [[ [ >integer number>string ] ]]
-fmt-o     = "o"                  => [[ [ >integer >oct ] ]]
-fmt-b     = "b"                  => [[ [ >integer >bin ] ]]
+fmt-d     = "d"                  => [[ [ number>string ] ]]
+fmt-o     = "o"                  => [[ [ >oct ] ]]
+fmt-b     = "b"                  => [[ [ >bin ] ]]
 fmt-e     = digits "e"           => [[ first '[ _ format-scientific ] ]]
```

Comment:
I did this because
```factor
1.5 "%x\n" printf
1.8p0
```

So I made "%d", "%o" and "%b" work like "%x".

Maybe we should do the other way around, add >integer to %x ?
In C, they use "%a" for hexadecimal floats. So if we want to look like C so that people are not surprised by the library, we can do this and drop binary and octal floats ?